### PR TITLE
fix: use lazy log formatting

### PR DESCRIPTION
**Pull request by kod-kristoff**
only format the log messages if the log level is enabled.

### CODE REVIEW

File: mink/core/jobs.py [hunk 1]
```diff
@@ -314,8 +316,9 @@ def uninstall_strix(self):
 
         if p.returncode != 0:
             stderr = p.stderr.decode() if p.stderr else ""
-            app.logger.error(f"Failed to uninstall corpus {self.id} from Strix: {stderr}")
-            raise exceptions.JobError(f"Failed to uninstall corpus from Strix: {stderr}")
```
Comment (MartinHammarstedt) [inline mink/core/jobs.py:318]: Is this removal intentional?

File: mink/core/jobs.py [hunk 2]
```diff
@@ -536,8 +542,11 @@ def clean_export(self):
 
         sparv_output = p.stdout.decode() if p.stdout else ""
         sparv_output = ", ".join([line for line in sparv_output.split("\n") if line])
-        if not ("Nothing to remove" in sparv_output or "'export' directory removed" in sparv_output):
```
Comment (MartinHammarstedt) [inline mink/core/jobs.py:539]: if-statement removed?

File: mink/metadata/metadata_routes.py [hunk 3]
```diff
@@ -93,17 +100,23 @@ def create_metadata(user: dict, auth_token: str):
...
         except Exception as err:
-            app.logger.error(f"Failed to remove corpus '{resource_id}' from auth system. {err}")
+            app.logger.error(
+                "Failed to remove corpus '%s' from auth system. %s", resource_id, err
+            )
         try:
             info_obj.remove()
         except Exception as err:
-            app.logger.error(f"Failed to remove object '{resource_id}' from registry. {err}")
-        return utils.response("Failed to create resource dir", err=True, info=str(e),
```
Comment (MartinHammarstedt) [inline mink/metadata/metadata_routes.py:105]: return removed

File: mink/sparv/storage_routes.py [hunk 4]
```diff
@@ -61,17 +62,21 @@ def create_corpus(user: dict, auth_token: str):
...
         except Exception as err:
-            app.logger.error(f"Failed to remove corpus '{resource_id}' from auth system. {err}")
+            app.logger.error(
+                "Failed to remove corpus '%s' from auth system. %s", resource_id, err
+            )
         try:
             info_obj.remove()
         except Exception as err:
-            app.logger.error(f"Failed to remove job '{resource_id}'. {err}")
-        return utils.response("Failed to create corpus dir", err=True, info=str(e),
```
Comment (MartinHammarstedt) [inline mink/sparv/storage_routes.py:73]: return removed

File: mink/sparv/storage_routes.py [hunk 5]
```diff
@@ -139,14 +144,14 @@ def remove_corpus(resource_id: str):
     try:
         info_obj.remove()
     except Exception as err:
-        app.logger.error(f"Failed to remove job '{resource_id}'. {err}")
-    return utils.response(f"Corpus '{resource_id}' successfully removed", return_code="removed_corpus")
```
Comment (MartinHammarstedt) [inline mink/sparv/storage_routes.py:143]: return removed

**Comment by kod-kristoff** [comment]
Sorry, I missed those lines!
vscode automatically ran ruff format, so I have sifted through the changes to only include those that corresponds to log changes...
All should be fix
